test/unit/testMemory.js

```diff
@@ -3,12 +3,12 @@ var proxyquire = require('proxyquire');
 var os = require('os');
 
 var mocks = {
-  './lib/sender': function() {
+  './lib/clients': function() {
     this.send = function() {};
   }
 };
 
-var metrics = proxyquire('index.js', mocks)({enabled: true});
+var metrics = proxyquire('index.js', mocks)({enabled: true, host: '127.0.0.1'});
```

Comment:
if no host is passed,  it will be treated as metrics disabled as there is no server to send the metrics data to. Before there just no such check. It should still be backward compatible.